nevergrad/optimization/recorded_recommendations.csv

```diff
@@ -119,7 +119,7 @@ QORecentering7ScrHammersleySearch,-0.968095889,0.0,0.3015091095,0.5891348635,,,,
 QOScrHammersleySearch,-1.3829941271,0.0,0.4307272993,0.8416212336,,,,,,,,,,,,
 QRBO,-1.0,0.0,0.5773502692,1.3763819205,,,,,,,,,,,,
 QrDE,-1.2126139659,-0.1022395502,0.5946113194,0.5365857957,3.38269024,1.3718912761,-3.2278357447,-1.2566858857,,,,,,,,
-RBO,3.9195896894,1.2677618652,1.1319508593,3.3209335263,,,,,,,,,,,,
+RBO,3.9195896209,1.2677618285,1.1319508564,3.3209334767,,,,,,,,,,,,
```

Comment:
This happens frequently, depending on the machine.
The test passes in spite of the difference precisely because we compare an adapted number of digits. The recorded number changes not because that test did not pass but because a new method was added so that the file was regenerated.